src/nupic/algorithms/knn_classifier.py

```diff
@@ -422,9 +422,9 @@ def learn(self, inputPattern, inputCategory, partitionId=None, isSparse=0,
         within distThreshold, only the first partitionId will be stored). This
         is an optional parameter.
 
-    :param isSparse: (int) If 0, the input pattern is a dense representation. If
-        isSparse > 0, the input pattern is a list of non-zero indices and
-        isSparse is the length of the dense representation
+    :param isSparse: (int) If 0, the input pattern is a dense representation.
+        If isSparse > 0, the input pattern is a list of non-zero indices of
+        the active bits and isSparse is the number of total bits (n).
```

Comment:
What do you mean by number of total bits? The number of ON bits or the length of the equivalent dense vector? Maybe good to give a small example.